jetty-osgi/jetty-osgi-boot/src/main/java/org/eclipse/jetty/osgi/boot/internal/serverfactory/ServerInstanceWrapper.java

```diff
@@ -142,7 +143,7 @@ public static Server configure(Server server, List<URL> jettyConfigurations, Dic
 
         for (URL jettyConfiguration : jettyConfigurations)
         {
-            try(Resource r = Resource.newResource(jettyConfiguration))
+            try(Resource r = new BundleResource(jettyConfiguration))
```

Comment:
This isn't sufficient. A few lines further down new XmlConfiguration(resource.getURI().toURL()) is called, which will have the same problem.

Reply:
Good point.
Will address this.